aip/general/0122.md

```diff
@@ -119,8 +119,9 @@ ID for the publisher, and `les-miserables` is the resource ID for the book.
     - Characters outside of ASCII **should not** be permitted; however, if
       Unicode characters are necessary, APIs **must** follow guidance in
       [AIP-210][].
-    - User-settable IDs **should not** be permitted to be a UUID (or any value
-      that syntactically appears to be a UUID).
+    - User-settable IDs **should not** be permitted to be a UUID or any value
+      that syntactically appears to be a UUID (anything matching 
+      /^[0-9a-f]{8}(-[0-9a-f]{4}){3}-[0-9a-f]{12}$/i).
```

Comment:
I think this change limits the requirements and modifies the intent.
Someone may decide to require for an id to be a string of 30
random characters. That would not pass your uuid validator
regex but would not be user friendly.